esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -743,6 +743,12 @@ def stats(prog):
             else:
                 vitals['studentnum'].append((key, students[key].filter(is_active = True).distinct().count()))
 
+
+        volunteer_list = []
+        volunteer_dict = prog.volunteers()
+        volunteer_list.append(("Volunteers who are signed up for at least one time slot", volunteer_dict['volunteer_all'].count()))
```

Comment:
This probably needs to be wrapped in some sort of check if the module is enabled at all -- such as `if 'volunteer_all' in volunteer_dict`.